src/components/10-atoms/fieldset/index.scss

```diff
@@ -37,6 +37,14 @@ axa-fieldset {
     }
   }
 
+  &[nostretchonmobile]{
+    @include breakpoint($mediaquery-xsm) {
+      axa-radio[button][nogap] {
+        width: auto !important;
```

Comment:
` !important` should be your your last choice.

Reply:
I think this is needed, because we have to override axa-radio from the fieldset component. But I will check that again.

Reply:
In my opinion `!important` is needed here. The specificity from the other selectors seems to be higher.

Reply:
I am also for alternatives, as long as they don't make the CSS ugly. If not possible, I guess we need it, but we should really try hard to not do that.

Reply:
Try to rewrite the selector for this line: `width: 100% !important;`. That line should not be selected if "nostretch" is set.